### PR TITLE
Migrate static breakpoint mixins

### LINKED ISSUE #7143: Migrate `page-content-when-fully-condensed` mixin
| Priority | Count | Notes |
| ----- | -----| ----- |
| High | 134 | Static value <br> [Reference breakpoints migration guide](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v9-to-v10.md#sass-static-mixins) |

### LINKED ISSUE #7144: Migrate `page-content-when-layout-stacked` mixin
| Priority | Count | Notes |
| ----- | -----| ----- |
| Low | 27 | Static value <br> [Reference breakpoints migration guide](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v9-to-v10.md#sass-static-mixins) |

### LINKED ISSUE #7145: Migrate `page-content-when-not-partially-condensed` mixin
| Priority | Count | Notes |
| ----- | -----| ----- |
| Medium | 50 | Static value <br> [Reference breakpoints migration guide](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v9-to-v10.md#sass-static-mixins) |

### LINKED ISSUE #7146: Migrate `page-content-when-partially-condensed` mixin
| Priority | Count | Notes |
| ----- | -----| ----- |
| High | 131 | Static value <br> [Reference breakpoints migration guide](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v9-to-v10.md#sass-static-mixins) |

### LINKED ISSUE #7147: Migrate `page-content-when-not-fully-condensed` mixin
| Priority | Count | Notes |
| ----- | -----| ----- |
| High | 103 | Static value <br> [Reference breakpoints migration guide](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v9-to-v10.md#sass-static-mixins) |

### PR DESCRIPTION
**Pull request by aaronccasanova**
Closes #7143, #7144, #7145, #7146, #7147

Add `replace-static-breakpoint-mixins` migration script.

Running `npx @shopify/polaris-migrator replace-static-breakpoint-mixins "**/*.scss"` will replace the following mixins with Polaris Breakpoint media queries:

| Before                                    | After                     |
| ----------------------------------------- | ------------------------- |
| page-content-when-partially-condensed     | #{$p-breakpoints-md-down} |
| page-content-when-not-partially-condensed | #{$p-breakpoints-md-up}   |
| page-content-when-fully-condensed         | #{$p-breakpoints-sm-down} |
| page-content-when-not-fully-condensed     | #{$p-breakpoints-sm-up}   |
| page-content-when-layout-stacked          | #{$p-breakpoints-md-down} |
| page-content-when-layout-not-stacked      | #{$p-breakpoints-md-up}   |
| page-before-resource-list-small           | #{$p-breakpoints-sm-down} |
| page-after-resource-list-small            | #{$p-breakpoints-sm-up}   |
| page-when-not-max-width                   | #{$p-breakpoints-lg-down} |
| when-typography-condensed                 | #{$p-breakpoints-md-down} |
| when-typography-not-condensed             | #{$p-breakpoints-md-up}   |
| frame-when-nav-hidden                     | #{$p-breakpoints-md-down} |
| frame-when-nav-displayed                  | #{$p-breakpoints-md-up}   |
| frame-with-nav-when-not-max-width         | #{$p-breakpoints-xl-down} |
| after-topbar-sheet                        | #{$p-breakpoints-sm-up}   |

### CODE REVIEW

File: .changeset/old-badgers-melt.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-migrator': minor
+---
+
+Add legacy Sass API migrations
```
Comment (sam-b-rose) [inline .changeset/old-badgers-melt.md]: Maybe specify here what migrations are being added (ex:  Add legacy Sass breakpoint migrations)

**Review by sam-b-rose — COMMENTED** [review]
Very clean! This should really help breakpoint things up 📱 💻 🖥️

**Review by sam-b-rose — APPROVED** [review]

File: .changeset/old-badgers-melt.md (hunk 1 above)
Comment (aaronccasanova) [reply .changeset/old-badgers-melt.md]: Changeset entry removed

**Pull request merged**